controller.py

```diff
@@ -62,13 +66,15 @@
     IdentifierResolutionRegistrar,
 )
 from canonicalize import AuthorNameCanonicalizer
+from integration_client import IntegrationClientCoverageProvider
 from problem_details import *
 
 HTTP_OK = 200
 HTTP_CREATED = 201
 HTTP_ACCEPTED = 202
 HTTP_UNAUTHORIZED = 401
 HTTP_NOT_FOUND = 404
+HTTP_UNPROCESSIBLE_ENTITY = 422
```

Comment:
I don't like using this HTTP code for metadata requests (especially since I think we use it for invalid URNs in the URNLookupController), but I couldn't find another http code that resonated. At the moment, I believe it's going to be ignored on the circulation manager anyway.

Reply:
Yeah, it's tough because this isn't even a real HTTP response. You could send 202 (which implies the server is still on the job) and since the caller knows that they asked for a list of items that it might be able to help with, it will know what 202 means in this context.